register.html.slim

```diff
@@ -8,7 +8,7 @@ head
       clientId: 'web'
     });
 
-    var redirectUri = 'http://developers.redhat.com/confirmation';
+    var redirectUri = app.ssoConfig.confirmation;
```

Comment:
@wesbos Any ideas on how I get the variables into this page? Currently I'm seeing 'Uncaught ReferenceError: app is not defined' when visiting this page.

I need to ensure the right KeyCloak server is used in staging.

I also need to paramaterise line 6 above (url: "https://developers.redhat.com/auth",)